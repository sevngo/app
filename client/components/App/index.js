import React from 'react';
import { Route, Switch, Redirect } from 'react-router-dom';
import Header from '../../containers/Header';
import Users from '../../containers/Users';
import User from '../../containers/User';
import Snackbars from '../../containers/Snackbars';
import Loading from '../../containers/Loading';
import Reset from '../../containers/Reset';
import Verify from '../../containers/Verify';
import { homeRoute, userRoute, resetRoute, verifyRoute } from '../../utils';
import useStyles from './styles';

const App = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Header />
      <Switch>
        <Route path={homeRoute} exact component={Users} />
        <Route path={userRoute(':id')} exact component={User} />
        <Route path={resetRoute(':token')} exact component={Reset} />
        <Route path={verifyRoute(':token')} exact component={Verify} />
        <Redirect to="/" />
      </Switch>
      <Snackbars />
      <Loading />
    </div>
  );
};

export default App;
