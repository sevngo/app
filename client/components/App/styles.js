export default {
  root: {
    backgroundImage: 'url(//www.gstatic.com/pantheon/images/marketplace/cameo_banner-1x.png)',
    minHeight: '100vh',
  },
};
