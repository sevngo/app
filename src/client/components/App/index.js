import React, { Suspense } from 'react';
import { Route, Switch, Redirect } from 'react-router-dom';
import { map } from 'ramda';
import { Box } from '@material-ui/core';
import Header from '../../containers/Header';
import Snackbar from '../../containers/Snackbar';
import Loader from '../../containers/Loader';
import PrivateRoute from '../PrivateRoute';
import UnPrivateRoute from '../UnPrivateRoute';
import useStyles from './styles';
import { routes, defaultRoute } from '../../utils';

const App = () => {
  const classes = useStyles();
  return (
    <Box className={classes.root}>
      <Header />
      <Loader />
      <Suspense fallback={<Loader />}>
        <Box p={3}>
          <Switch>
            {map((route) => {
              const { isPrivate, isUnPrivate, path } = route;
              if (isPrivate) return <PrivateRoute key={path} {...route} />;
              if (isUnPrivate) return <UnPrivateRoute key={path} {...route} />;
              return <Route key={path} {...route} />;
            })(routes)}
            <Redirect to={defaultRoute.path} />
          </Switch>
        </Box>
      </Suspense>
      <Snackbar />
    </Box>
  );
};

export default App;
