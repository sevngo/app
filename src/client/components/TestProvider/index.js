import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import { MuiThemeProvider } from '@material-ui/core';
import { IntlProvider } from 'react-intl';
import { theme } from '../../utils/theme';
import reducer from '../../reducers';
import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import defaultInitialState from './initialState';

jest.mock('../../store');

const TestProvider = ({ children, initialState = defaultInitialState }) => {
  const store = createStore(reducer, initialState, applyMiddleware(thunk));
  return (
    <Provider store={store}>
      <IntlProvider locale="en" messages={{}}>
        <BrowserRouter>
          <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>
        </BrowserRouter>
      </IntlProvider>
    </Provider>
  );
};

export default TestProvider;
