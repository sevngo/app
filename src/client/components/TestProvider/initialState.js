const initialState = {
  auth: {
    _id: '6008432b6ed92c2ac837ec6c',
    username: 'asdasd',
    email: 'asdasd@asdasd.com',
    gender: 'female',
    birthDate: '1991-01-20T00:00:00.000Z',
    address: {
      name: '5 All√©e Claude Chastillon, 93290 Tremblay-en-France, France',
      type: 'Point',
      coordinates: [2.5654428, 48.955157299999996],
    },
    notifications: [
      {
        user: {
          _id: '5caf003feaca23291c0cec34',
          username: 'qweqwe',
        },
        createdAt: '2020-01-23T14:10:55.857Z',
        _id: '5e29a96f7e701b58a77dcdbb',
        messageId: 'friendLogged',
      },
      {
        user: {
          _id: '5caf003feaca23291c0cec36',
          username: 'asdasd',
        },
        createdAt: '2020-01-23T14:11:55.857Z',
        _id: '5e29a96f7e701b58a77dcdba',
        messageId: 'friendLogged',
      },
    ],
    token:
      'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2MDA4NDMyYjZlZDkyYzJhYzgzN2VjNmMiLCJpYXQiOjE2MTEyNTc1MjQsImV4cCI6MTYxMTM0MzkyNH0.suG2IyrXwsI3bOcK1jG7XpasHfwQM1Fip3YrUzX01Pc',
  },
  users: {
    filter: {
      gender: 'male',
      maxDistance: 20000,
      ageRange: [18, 50],
      sortBy: 'distance:asc',
      limit: 10,
      skip: 0,
    },
    data: [
      {
        _id: '60084a46203c4e342b13114c',
        username: 'Shanie21',
        birthDate: '1998-03-03T13:43:52.868Z',
        gender: 'male',
        address: {
          name: '247 Bert Village',
        },
        distance: 2640.691228895585,
      },
      {
        _id: '60084a46203c4e342b13113c',
        username: 'Brianne82',
        birthDate: '1990-05-11T19:00:30.586Z',
        gender: 'male',
        address: {
          name: '4860 Berenice Lodge',
        },
        distance: 2993.4424127582033,
      },
      {
        _id: '60084a46203c4e342b131140',
        username: 'Jon3',
        birthDate: '1973-05-08T21:14:45.718Z',
        gender: 'male',
        address: {
          name: '259 Fabiola Station',
        },
        distance: 6966.4502984182745,
      },
      {
        _id: '60084a46203c4e342b131139',
        username: 'Joannie_Corkery92',
        birthDate: '1998-09-22T09:23:53.485Z',
        gender: 'male',
        address: {
          name: '2431 Marisol Park',
        },
        distance: 9156.496213903085,
      },
      {
        _id: '60084a46203c4e342b13113a',
        username: 'Lilly.MacGyver',
        birthDate: '2002-05-18T07:47:56.337Z',
        gender: 'male',
        address: {
          name: '27384 Carroll Rue',
        },
        distance: 10229.485636305038,
      },
      {
        _id: '60084a46203c4e342b131143',
        username: 'Lacy_Harber23',
        birthDate: '2001-11-18T21:39:20.224Z',
        gender: 'male',
        address: {
          name: '1810 Eliseo Key',
        },
        distance: 11798.426998259116,
      },
      {
        _id: '60084a46203c4e342b131147',
        username: 'Sophie.Gorczany',
        birthDate: '1992-08-23T18:55:03.359Z',
        gender: 'male',
        address: {
          name: '149 Koch Vista',
        },
        distance: 13149.980680446492,
      },
      {
        _id: '60084a46203c4e342b131142',
        username: 'Joelle.Weissnat',
        birthDate: '1985-02-24T06:13:19.138Z',
        gender: 'male',
        address: {
          name: '74795 Howe Inlet',
        },
        distance: 16796.075505426317,
      },
    ],
    total: 8,
  },
  user: {
    _id: '60084a46203c4e342b13114c',
    username: 'Shanie21',
    birthDate: '1998-03-03T13:43:52.868Z',
    gender: 'male',
    address: {
      name: '247 Bert Village',
    },
  },
  loading: true,
  snackbar: {
    variant: 'success',
  },
};

export default initialState;
