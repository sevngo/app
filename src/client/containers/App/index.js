import React, { Suspense } from 'react';
import { Route, Routes, Navigate } from 'react-router-dom';
import { map } from 'ramda';
import { Box } from '@material-ui/core';
import Header from '../Header';
import Snackbar from '../Snackbar';
import Loader from '../Loader';
import PrivateRoute from '../../components/routing/PrivateRoute';
import UnPrivateRoute from '../../components/routing/UnPrivateRoute';
import useStyles from './styles';
import { routes, defaultRoute } from '../../utils';

const App = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Header />
      <Loader />
      <Suspense fallback={<Loader />}>
        <Box p={3}>
          <Routes>
            {map((route) => {
              const { isPrivate, isUnPrivate, Component, ...rest } = route;
              if (isPrivate)
                return (
                  <Route
                    element={
                      <PrivateRoute>
                        <Component />
                      </PrivateRoute>
                    }
                    {...rest}
                  />
                );
              if (isUnPrivate)
                return (
                  <Route
                    element={
                      <UnPrivateRoute>
                        <Component />
                      </UnPrivateRoute>
                    }
                    {...rest}
                  />
                );
              return <Route element={<Component />} {...rest} />;
            })(routes)}
            <Navigate to={defaultRoute.path} />
          </Routes>
        </Box>
      </Suspense>
      <Snackbar />
    </div>
  );
};

export default App;
