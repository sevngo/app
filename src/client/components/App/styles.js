import { makeStyles } from '@material-ui/core/styles';

export default makeStyles({
  root: {
    backgroundImage:
      'url(//www.gstatic.com/pantheon/images/marketplace/cameo_banner-1x.png)',
    minHeight: '100vh',
  },
});
