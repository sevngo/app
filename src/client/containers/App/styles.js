import { makeStyles } from '@material-ui/core/styles';

export default makeStyles({
  root: {
    backgroundImage:
      'url(//www.gstatic.com/pantheon/images/marketplace/cameo_banner-1x.png)',
    minHeight: '100vh',
  },
  pointerEventsNone: { pointerEvents: 'none' },
  pointerEventsAuto: { pointerEvents: 'auto' },
  visibilityHidden: { visibility: 'hidden' },
  visibilityVisible: { visibility: 'visible' },
});
