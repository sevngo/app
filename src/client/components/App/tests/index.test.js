import React from 'react';
import { render } from '@testing-library/react';
import TestProvider from '../../../components/TestProvider';
import App from '../index';

describe('App', () => {
  const Component = (
    <TestProvider>
      <App />
    </TestProvider>
  );
  it('should render without crash', () => {
    const { container } = render(Component);
    expect(container).toBeDefined();
  });
});
