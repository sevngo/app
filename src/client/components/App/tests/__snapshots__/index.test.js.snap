// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-2 makeStyles-root-1"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-appBar-3 MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-icon-5 MuiIconButton-colorInherit"
          href="/"
          role="button"
          tabindex="0"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
        <h6
          class="MuiTypography-root makeStyles-title-4 MuiTypography-h6 MuiTypography-colorInherit"
        >
          Matcha
        </h6>
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <span
              class="MuiBadge-root"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
                />
              </svg>
              <span
                class="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorError"
              >
                2
              </span>
            </span>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
          data-testid="accountButton"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </header>
    <div
      class="MuiLinearProgress-root MuiLinearProgress-colorPrimary makeStyles-visibilityVisible-8 MuiLinearProgress-indeterminate"
      role="progressbar"
      thickness="8"
    >
      <div
        class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate"
      />
      <div
        class="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
      />
    </div>
    <div
      class="MuiBox-root MuiBox-root-9"
    >
      <div
        class="MuiPaper-root makeStyles-paper-12 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h6v-2H3v2zM3 6v2h18V6H3zm0 7h12v-2H3v2z"
              />
            </svg>
            <p
              class="MuiTypography-root makeStyles-filter-13 MuiTypography-body1"
            >
              Filter
            </p>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <div
          class="MuiTablePagination-root makeStyles-pagination-11"
        >
          <div
            class="MuiToolbar-root MuiToolbar-regular MuiTablePagination-toolbar MuiToolbar-gutters"
          >
            <div
              class="MuiTablePagination-spacer"
            />
            <p
              class="MuiTypography-root MuiTablePagination-caption MuiTypography-body2 MuiTypography-colorInherit"
              id="mui-99862"
            >
              Rows per page:
            </p>
            <div
              class="MuiInputBase-root MuiTablePagination-input makeStyles-displayNone-10 MuiTablePagination-selectRoot"
            >
              <div
                aria-haspopup="listbox"
                aria-labelledby="mui-99862 mui-56074"
                class="MuiSelect-root MuiSelect-select MuiTablePagination-select MuiSelect-selectMenu MuiInputBase-input"
                id="mui-56074"
                role="button"
                tabindex="0"
              >
                10
              </div>
              <input
                aria-hidden="true"
                class="MuiSelect-nativeInput"
                tabindex="-1"
                value="10"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSelect-icon MuiTablePagination-selectIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
            <p
              class="MuiTypography-root MuiTablePagination-caption MuiTypography-body2 MuiTypography-colorInherit"
            >
              1-8 of 8
            </p>
            <div
              class="MuiTablePagination-actions"
            >
              <button
                aria-label="Previous page"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                title="Previous page"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                    />
                  </svg>
                </span>
              </button>
              <button
                aria-label="Next page"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                title="Next page"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root MuiBox-root-14"
      />
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b13114c"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Shanie21
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    22
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  2640 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 133ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b13113c"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Brianne82
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    30
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  2993 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 400ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 266ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b131140"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Jon3
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    47
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  6966 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 600ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 400ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b131139"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Joannie_Corkery92
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    22
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  9156 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 800ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 533ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b13113a"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Lilly.MacGyver
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    18
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  10229 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 1000ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 666ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b131143"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Lacy_Harber23
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    19
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  11798 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 1200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 799ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b131147"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Sophie.Gorczany
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    28
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  13149 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
          style="opacity: 1; transform: scale(1, 1); transition: opacity 1400ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 932ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiCardActionArea-root"
              href="/user/60084a46203c4e342b131142"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiCardMedia-root makeStyles-media-15"
                style="background-image: url(emptyImage.png);"
              />
              <div
                class="MuiCardContent-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <h5
                    class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
                  >
                    Joelle.Weissnat
                  </h5>
                  <h5
                    class="MuiTypography-root makeStyles-age-16 MuiTypography-h5"
                  >
                    , 
                    35
                  </h5>
                </div>
                <p
                  class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                >
                  16796 km
                </p>
              </div>
              <span
                class="MuiCardActionArea-focusHighlight"
              />
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiSnackbar-root MuiSnackbar-anchorOriginBottomLeft"
    >
      <div
        class="MuiPaper-root MuiSnackbarContent-root makeStyles-success-18 MuiPaper-elevation6"
        direction="up"
        role="alert"
        style="opacity: 1; transform: scale(1, 1); transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      >
        <div
          class="MuiSnackbarContent-message"
        >
          <p
            class="MuiTypography-root makeStyles-message-24 MuiTypography-body2"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-iconVariant-23"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
              />
            </svg>
          </p>
        </div>
        <div
          class="MuiSnackbarContent-action"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
